onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -84,6 +115,8 @@ def append_to_tableau_colulmn_headers(header, question_type=None):
                 'dataType': quest_type,
                 'alias': header
             })
+        # Remove metadata fields from the column headers
+        remove_metadata_fields(self.xform_headers)
```

Comment:
Should this be:

```python
self.xform_headers = remove_metadata_fields(self.xform_headers)
```
Or better yet

```python
xform_headers = remove_metadata_fields(self.xform_headers)
...
for header in xform_headers:
    ...
```
The function `remove_metadata_fields()` does return data.
```

Reply:
yeah....changed that. Thanks for that.